volmdlr/edges.py

```diff
@@ -4376,7 +4376,7 @@ def revolution(self, axis_point: volmdlr.Point3D, axis: volmdlr.Vector3D,
             return [surface.rectangular_cut(0, angle,
                                             arc2d.angle1, arc2d.angle2)]
 
-    def to_step(self, current_id):
+    def to_step(self, current_id, **kwargs):
```

Comment:
There are no `kwargs`, I think !

Reply:
I have added **kwargs to all to_step methods, so we follow a standard. What do you think?